tests/test-sources/plugins/utils/web-devicons.nix

```diff
@@ -0,0 +1,22 @@
+{
+  empty = {
+    plugins.web-devicons.enable = true;
+  };
+
+  default = {
+    plugins.web-devicons = {
+      enable = true;
+    };
+  };
```

Comment:
If this is identical to `empty`, there's no point including it really.

If you want to test some known defaults, that's fine though, even if we aren't declaring nix options for them.